plan/logical_plan_builder.go

```diff
@@ -115,7 +115,9 @@ func (b *planBuilder) buildAggregation(p LogicalPlan, aggFuncList []*ast.Aggrega
 	for _, col := range p.Schema().Columns {
 		newFunc := aggregation.NewAggFuncDesc(b.ctx, ast.AggFuncFirstRow, []expression.Expression{col}, false)
 		plan4Agg.AggFuncs = append(plan4Agg.AggFuncs, newFunc)
-		schema4Agg.Append(col)
+		newCol, _ := col.Clone().(*expression.Column)
```

Comment:
So one column occurs in agg and agg' child. The tp of this two is different?

Reply:
Possibly. Actually, they are two columns if the type inferred after `NewAggFuncDesc()` is different from the origin type.

Reply:
Since they are different, why `unique id` is the same one.